backend/wallets/tests/wallet/test_WalletControllers.py

```diff
@@ -0,0 +1,371 @@
+from ast import List
+from math import exp
+import pytest
+from unittest.mock import patch, MagicMock
+
+from django.conf import settings
+
+from wallets.models import Wallet
+from wallets.controllers.wallet import WalletController
+from wallets.schema.wallet import (
+    BuildWalletRequest,
+    BuildWalletResponse,
+    ListWalletsRequest,
+    ListWalletsResponse,
+    WalletDetailsResponse,
+    WalletDetailsRequest,
+    UpdateWalletRequest,
+    UpdateWalletResponse,
+    DeleteWalletRequest,
+    DeleteWalletResponse,
+)
+
+from wallets.tests.test_fixture import test_user
+from wallets.tests.wallet.test_fixture import single_wallet
+
+
+@pytest.mark.django_db
+@patch("wallets.repository.wallet.WalletRepository.create_wallet")
+@patch("wallets.repository.wallet.WalletRepository.add_co_owners_to_wallet")
+def test_build_wallet_with_co_owner(
+    add_co_owners_to_wallet_mock, mock_create_wallet, test_user, single_wallet
+):
+    mock_wallet = single_wallet
+    mock_wallet.co_owners = [test_user[1]]
```

Comment:
Nitpick: Avoid using magic numbers! Make it explicit what test_user[1] means. e.g.
```
user_one = test_user[1]
```
Not an important thing but helps readability. PS: test_user is probably not a good naming for this fixture since it returns an iterable.